examples/catch-all-routes/pages/post/[...slug].js

```diff
@@ -0,0 +1,16 @@
+import { useRouter } from 'next/router'
+import Header from '../../components/header'
+
+const Comment = () => {
+  const router = useRouter()
+  const slug = router.query.slug || []
```

Comment:
This cannot be null:
```suggestion
  const slug = router.query.slug
```

Reply:
It's `null` in the first render, I thought that was expected because it doesn't have `getInitialProps` 🤔

Reply:
Oh, I forgot about automatic static prerendering.